edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Microsoft.Azure.Devices.Edge.Hub.Service.csproj

```diff
@@ -42,6 +42,8 @@
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="2.2.0" />
 	<PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.3.1" />
+	<PackageReference Include="prometheus-net" Version="3.4.0" />
```

Comment:
I would like to not take a dependency on the prometheus-net lib from Service

Reply:
spoke in person: 
prometheus-net.AspNetCore has a dependency on both prometheus-net and aspnet core. Either edgeHub service must take a dependency on Prometheus, or Util must take a dependency on asp.net. 

We decided service should take the dependency.